### DISCUSSION TITLE
demande d'aide pour aveyo et toute la communauté

### DISCUSSION THREAD

**Discussion by klink270** [AveYo/MediaCreationTool.bat#60] [General]
bonjour , je vous écrit car je suis pas expert du tout j'aimerais apprendre a utiliser correctement GITHUB pouvez vous m'aider juste a commencer un projet facile et évoluer ensemble je suis très intéresser par l'informatique , je monte des pc jusque la tout va c'est quand il y a des problème que je suis pas très fort merci d'avance même si personne ne veux j'aimerais juste bien comprendre alors github car sa me parait un très bonne outil .

bien à vous

**Comment by AveYo** [comment]
L'interface graphique Web github est assez simple pour un projet de démarrage, 
vous pouvez faire presque tout dans le navigateur Web - ajouter des fichiers, éditer des fichiers, etc.
Le formatage du texte, des images, des liens est Markdown ou HTML - Google est ton meilleur ami ;)
Je suggère fortement de rechercher des guides ou simplement de regarder quelques tutoriels youtube.
Bonne chance!
